tests/integration/models/reddit/test_submission.py

```diff
@@ -537,3 +537,19 @@ def test_unspoiler(self):
         self.reddit.read_only = False
         with self.use_cassette():
             Submission(self.reddit, "5ouli3").mod.unspoiler()
+
+    @mock.patch("time.sleep", return_value=None)
+    def test_update_crowd_control_level(self, _):
+        self.reddit.read_only = False
+        submission = self.reddit.submission("4s2idz")
+        with self.use_cassette():
+            self.reddit.submission("4s2idz").mod.update_crowd_control_level(2)
+            modlog = next(
+                iter(
+                    self.reddit.subreddit("mod").mod.log(
+                        action="adjust_post_crowd_control_level", limit=1
+                    )
+                )
+            )
+            assert modlog.action == "adjust_post_crowd_control_level"
+            assert modlog.target_fullname == submission.fullname
```

Comment:
```suggestion
            assert modlog.target_fullname == submission.fullname
            assert modlog.details == "medium"
```
Can we check that it actually set it to the correct level?